File: app/src/main/java/com/unpas/masteringmaths/student/activity/StudentResultExamActivity.java
package com.unpas.masteringmaths.student.activity;

import android.annotation.SuppressLint;
import android.content.pm.ActivityInfo;
import android.graphics.Color;
import android.os.Bundle;
import android.view.Menu;
import android.view.MenuItem;
import android.widget.TextView;
import androidx.annotation.NonNull;
import androidx.appcompat.app.AppCompatActivity;
import androidx.appcompat.widget.Toolbar;
import com.unpas.masteringmaths.R;
import com.unpas.masteringmaths.teacher.model.Teacher;

import static com.unpas.masteringmaths.student.activity.StudentExamActivity.*;

public class StudentResultExamActivity extends AppCompatActivity {

    private int myPoint;
    private int myTrueAnswer;
    private int myFalseAnswer;
    private TextView tvTrue, tvFalse, tvPoint, tvBad, tvGood;
    private Toolbar toolbar;

    @SuppressLint("SetTextI18n")
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_student_result_exam);
        setRequestedOrientation(ActivityInfo.SCREEN_ORIENTATION_PORTRAIT);

        toolbar = findViewById(R.id.toolbar);
        setSupportActionBar(toolbar);

        tvTrue = findViewById(R.id.tv_true);
        tvFalse = findViewById(R.id.tv_false);
        tvPoint = findViewById(R.id.tv_myscore);
        tvBad = findViewById(R.id.bad);
        tvGood = findViewById(R.id.good);

        myPoint = getIntent().getIntExtra(MY_POINT, 0);
        myTrueAnswer = getIntent().getIntExtra(TRUE_ANSWER, 0);
        myFalseAnswer = getIntent().getIntExtra(FALSE_ANSWER, 0);

        if (myPoint >= 70) {
            tvGood.setBackgroundColor(Color.parseColor("#FF15D600"));
            tvGood.setTextColor(Color.parseColor("#FFFFFF"));
        }else {
            tvBad.setBackgroundColor(Color.parseColor("#FFF60400"));
            tvBad.setTextColor(Color.parseColor("#FFFFFF"));
        }

        tvTrue.setText("Benar : " + myTrueAnswer);
        tvFalse.setText("Salah : " + myFalseAnswer);
        tvPoint.setText(String.valueOf(myPoint));

    }

    @Override
    public boolean onCreateOptionsMenu(Menu menu) {
        getMenuInflater().inflate(R.menu.result_activity,menu);
        return true;
    }

    @Override
    public boolean onOptionsItemSelected(@NonNull MenuItem item) {
        if (item.getItemId() == R.id.finish) {
            finish();
        }
        return true;
    }
}

File: app/src/main/java/com/unpas/masteringmaths/student/activity/StudentResultExamActivity.kt
package com.unpas.masteringmaths.student.activity

import android.annotation.SuppressLint
import android.content.pm.ActivityInfo
import android.graphics.Color
import android.os.Bundle
import android.view.Menu
import android.view.MenuItem
import androidx.appcompat.app.AppCompatActivity
import com.unpas.masteringmaths.R
import com.unpas.masteringmaths.main.GlideApp
import kotlinx.android.synthetic.main.activity_student_result_exam.*

class StudentResultExamActivity : AppCompatActivity() {

    private var myPoint = 0
    private var myTrueAnswer = 0
    private var myFalseAnswer = 0

    @SuppressLint("SetTextI18n")
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_student_result_exam)
        init()

        if (myPoint >= 70) {
            GlideApp.with(this)
                .load(R.drawable.trophy_icon)
                .into(img_result)

            tv_congratulation.text = getString(R.string.tv_congratulation)
            tv_congratulation.setTextColor(Color.parseColor("#2ECC71"))
        } else {
            GlideApp.with(this)
                .load(R.drawable.remedial)
                .into(img_result)

            tv_congratulation.text = getString(R.string.tv_exam_failed)
            tv_congratulation.setTextColor(Color.parseColor("#F44336"))
        }

        tv_true.text = "Correct Answe : $myTrueAnswer"
        tv_false.text = "Wrong Answer : $myFalseAnswer"
        tv_myscore.text = myPoint.toString()
    }

    private fun init() {
        setSupportActionBar(toolbar)
        requestedOrientation = ActivityInfo.SCREEN_ORIENTATION_PORTRAIT
        myPoint = intent.getIntExtra(StudentExamActivity.MY_POINT, 0)
        myTrueAnswer = intent.getIntExtra(StudentExamActivity.TRUE_ANSWER, 0)
        myFalseAnswer = intent.getIntExtra(StudentExamActivity.FALSE_ANSWER, 0)
    }

    override fun onCreateOptionsMenu(menu: Menu): Boolean {
        menuInflater.inflate(R.menu.result_activity, menu)
        return true
    }

    override fun onOptionsItemSelected(item: MenuItem): Boolean {
        if (item.itemId == R.id.finish) {
            finish()
        }
        return true
    }
}